source/whats-new.txt

```diff
@@ -42,12 +42,30 @@ Upcoming Breaking Changes
 What's New in 4.10
 ------------------
 
+.. important:: Aggregation Expression Operations are in Beta
+   
+   Version 4.9 of the {+driver-short+} introduced an API for creating
+   :ref:`aggregation expressions <java-aggregation-expression-operations>`. 
+   This feature is still in beta phase.
```

Comment:
Did anything change between 4.9 and 4.10? Wondering if we need to call out that it's still in beta.

Reply:
I'm going to move this note to v4.9